Packs/FireEyeHX/Playbooks/playbook-FireEye_HX_-_Traffic_Indicators_Hunting.yml

```diff
@@ -0,0 +1,1311 @@
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "152"
+    scriptarguments:
+      append:
+        simple: "true"
+      key:
+        simple: FireEyeHX.HuntingResults
+      value:
+        complex:
+          root: HuntingResults3
+          filters:
+          - - operator: notInList
+              left:
+                value:
+                  simple: HuntingResults3.id
+                iscontext: true
+              right:
+                value:
+                  simple: FireEyeHX.HuntingResults.id
+                iscontext: true
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 3280,
+          "y": 600
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {
+      "109_136_#default#": 0.14,
+      "114_152_#default#": 0.14
+    },
+    "paper": {
+      "dimensions": {
+        "height": 1335,
+        "width": 2900,
+        "x": 960,
+        "y": -360
+      }
+    }
+  }
+inputs:
+- key: IPAddress
+  value: {}
+  required: false
+  description: |-
+    A single or multiple IP address to search for in FireEye HX logs. Used for both source and destination IP addresses.
+
+    Separate multiple search values by commas only (without spaces or any special characters).
+  playbookInputQuery:
+- key: PortNumber
+  value: {}
+  required: false
+  description: |-
+    A single or multiple port numbers to search for in FireEye HX logs. Used for both remote and local ports.
+
+    Separate multiple search values by commas only (without spaces or any special characters).
+  playbookInputQuery:
+- key: URLDomain
+  value: {}
+  required: false
+  description: |-
+    Single or multiple URLs and/or domains to search for in FireEye HX logs.
+    By default, the 'contains' clause is used.
+
+    Separate multiple search values by commas only (without spaces or any special characters).
+  playbookInputQuery:
+- key: hostSetName
+  value: {}
+  required: true
+  description: The name of host set to be searched.
+  playbookInputQuery:
+- key: interval_in_seconds
+  value: {}
+  required: false
+  description: The interval in seconds between each poll. Default is 60.
+  playbookInputQuery:
+- key: limit
+  value: {}
+  required: false
+  description: Limits the results count. (When the limit is reached, the search stops).
+  playbookInputQuery:
+- key: exhaustive
+  value:
+    simple: "no"
+  required: false
+  description: |-
+    Whether a search is exhaustive.
+    Possible values are: yes, no. Default is yes.
+  playbookInputQuery:
+outputs:
+- contextPath: FireEyeHX.HuntingResults
+  description: Email message objects and fields that were retrieved from FireEye HX.
+  type: string
+- contextPath: FireEyeHX.HuntingResults.Timestamp - Event
```

Comment:
Should contextPath names have spaces?  Please check all contextPaths.

Reply:
These are the outputs of the 'fireeye-hx-search (FireEyeHX v2)' command which contains spaces. 

![Screenshot 2023-01-15 at 10 38 32](https://user-images.githubusercontent.com/112805149/212531191-80a44a3d-97ea-4882-89cf-5b7d162bf1ab.png)